__docs__/wonder-blocks-form/checkbox.stories.tsx

```diff
@@ -53,9 +61,10 @@ Controlled.parameters = {
         disableSnapshot: true,
     },
     docs: {
-        description: {
-            story: "Use state to keep track of whether the checkbox is checked or not",
-        },
+        description: `Use state to keep track of whether the checkbox
```

Comment:
**suggestion:** `description.story` is needed so Storybook displays the description next to the example. You can see how it's missing here: 

https://5e1bf4b385e3fb0020b7073c-nwtpfilbpc.chromatic.com/?path=/docs/form-checkbox--docs#controlled